### PR TITLE
feat: add command and response bot scaffold/template

### PR DESCRIPTION
**Pull request by a1exwang**
- add command and response bot template from [this repo main branch](https://github.com/dooriya/commandResponseBot/tree/main)
- add scaffold UI

manifest changes and bot plugin settings capabilities are in later PRs.

### Scaffold UI
image

### Scaffolded files
image

### Command and response bot local debug
img
 
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/13883569

### CODE REVIEW

**Comment by a1exwang** [comment]
E2E TEST: n/a

File: packages/fx-core/src/plugins/solution/fx-solution/question.ts [hunk 1]
```diff
@@ -35,6 +35,14 @@ export const NotificationOptionItem: OptionItem = {
   detail: getLocalizedString("core.NotificationOption.detail"),
 };
 
+export const CommandAndResponseOptionItem: OptionItem = {
+  id: "CommandAndResponse",
+  label: "Command and Response",
```
Comment (dooriya) [inline packages/fx-core/src/plugins/solution/fx-solution/question.ts:40]: Shall we call it `Command and Response Bot` here to let users know they're going to build a Teams Bot?

Comment (a1exwang) [reply packages/fx-core/src/plugins/solution/fx-solution/question.ts]: Sure, updated.

**Review by dooriya — APPROVED** [review]

**Review by Siglud — APPROVED** [review]
LGTM

**Review by jayzhang — APPROVED** [review]

**Pull request merged**